did/model.go

```diff
@@ -1,10 +1,10 @@
 package did
 
 const (
-	LightDIDMode int = iota
-	BasicDIDMode
+	BasicDIDMode int = iota
 	StandardDIDMode
 	WalletDIDMode
+	LightDIDMode
```

Comment:
Could have added as option 4 rather than 3 . It works fine now since we dont usually make wallets with child. But this might open some corner case issues during switch or if/else statements.

Reply:
In that case I will change the sequence again.